src/local_scheduler/local_scheduler.cc

```diff
@@ -1426,6 +1427,8 @@ void start_server(
  * suite has its own declaration of main. */
 #ifndef LOCAL_SCHEDULER_TEST
 int main(int argc, char *argv[]) {
+  RayLog::StartRayLog(argv[0], RAY_INFO);
+  ray::SignalHandlers::InstallSingalHandler(argv[0], false);
```

Comment:
Use RAII to avoid missing UninstallSingalHandler call.
Create a constructor/deconstructor for SignalHandlers, in the constructor call InstallSingalHandler and call UninstallSingalHandler in the deconstructor.

Reply:
So does for the RayLog